CollAction/Controllers/GraphQlController.cs

```diff
@@ -140,7 +140,7 @@ private async Task<ExecutionResult> Execute(string query, string? operationName,
                 UserContext = new UserContext(User, context, serviceProvider),
                 ComplexityConfiguration = new ComplexityConfiguration()
                 {
-                    MaxDepth = 20
+                    MaxDepth = 21
```

Comment:
Magic number? Why is it 21 now? What does this mean? Use Constant With meaningful name

Reply:
Should be fixed